src/frame/io.rs

```diff
@@ -232,7 +232,7 @@ mod tests {
     #[test]
     fn encode_decode_identity() {
         fn property(f: Frame<()>) -> bool {
-            async_std::task::block_on(async move {
+            futures::executor::block_on(async move {
```

Comment:
Why use `futures::executor` here and `Runtime::block_on` in other places?

Reply:
The correctness does not depend on the runtime choice so it does not really matter. As this is a small property test without concurrency, executing it on the same thread is more light-weight.